apps/web/src/pages/Signin/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { signIn } from 'next-auth/react';
+import { StyledSignin, LoginButton, LoginContent, Header } from './styles';
+
+interface SigninProps {
+  providers: { id: string; name: string }[];
+}
+
+const Signin = ({ providers }: SigninProps) => {
+  const onSubmit = (data) => alert(JSON.stringify(data, null, 2));
```

Comment:
this is not being used?

Reply:
I commented out the onSubmit line.

Reply:
ah ok, could you remove then? if it's not being used.